<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	font: 22px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font:30px;
	color:#444444;
}</string>
    </property>
    <property name="text">
     <string>实验一：数字波形生成</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>逻辑输入</string>
    </property>
    <widget class="QLineEdit" name="data_A">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数字序列A：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数字序列B：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="data_B">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>逻辑函数F：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="data_F">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开始运算</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>数据重置</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>运算输出</string>
    </property>
    <widget class="QTextBrowser" name="res_console">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>721</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>波形显示</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>691</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="fig_con"/>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
